src/pages/iou/IOUModal.js

```diff
@@ -82,6 +95,14 @@ class IOUModal extends Component {
         }));
     }
 
+    handleParticipants(userEmails) {
+        this.setState(prevState => ({
```

Comment:
Do you need to use a function for `setState` here? I think you could just do:

```javascript
this.setState(userEmails);
```
And that should only override `this.state.userEmails` and leave the other state fields alone.